doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -340,7 +340,7 @@ Refer to [this Grafana document](https://grafana.com/tutorials/run-grafana-behin
   * Click "Import".
   * Click "Upload JSON file".
   * Choose a JSON file.
-    * Case 1: If you are using Ray 2.5.0, you can use [config/grafana/default_grafana_dashboard.json](https://github.com/ray-project/kuberay/blob/master/config/grafana/default_grafana_dashboard.json).
+    * Case 1: If you are using Ray 2.9.0, you can use [the sample config files on GitHub repo](https://github.com/ray-project/kuberay/tree/master/config/grafana) (those files that are named `xxx_grafana_dashboard.json`.
     * Case 2: Otherwise, you should import the `default_grafana_dashboard.json` file from `/tmp/ray/session_latest/metrics/grafana/dashboards/` in the head Pod. You can use `kubectl cp` to copy the file from the head Pod to your local machine.
```

Comment:
You also need to update "Case 2". Currently, the head Pod not only generates `default_grafana_dashboard.json` but also JSON files for Ray Data and Ray Serve Grafana dashboards.

Reply:
I will ping our Ray doc team later after you address this.

Reply:
Done